docs/hub/adding-a-task.md

```diff
@@ -0,0 +1,78 @@
+---
+title: Adding a task
+---
+
+# Adding a task to the Hub
+
+## Overview
```

Comment:
should we start with a "What's a task" header with basically this screenshot?

<img width="761" alt="Screenshot 2021-09-08 at 17 43 27" src="https://user-images.githubusercontent.com/326577/132541460-58a97cbe-7898-4637-9db3-41ecedd2ef04.png">

Reply:
Yes, done and moved things around!